topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -281,10 +302,10 @@ We will map our RNA reads to the *Drosophila melanogaster* genome using STAR.
 
 > ### {% icon hands_on %} Hands-on: Spliced mapping
 >
-> 1. Import the Ensembl gene annotation for *Drosophila melanogaster* (`Drosophila_melanogaster.BDGP6.87.gtf`) from the Shared Data library if available or from [Zenodo](https://zenodo.org/record/1185122/files/Drosophila_melanogaster.BDGP6.87.gtf) into your current Galaxy history
+> 1. Import the Ensembl gene annotation for *Drosophila melanogaster* (`Drosophila_melanogaster.BDGP6.87.gtf`) from the Shared Data library if available or from [Zenodo]({{ page.zenodo_link }}/files/Drosophila_melanogaster.BDGP6.87.gtf) into your current Galaxy history
```

Comment:
For the shared data library, it may change from one Galaxy instance to the other. Usually we say it during the workshop. I mention that it could be available on the shared data but without giving any path, but also from UCSC